src/main/java/redis/clients/jedis/BinaryClient.java

```diff
@@ -421,8 +424,7 @@ public void zadd(final byte[] key, final Map<byte[], Double> scoreMembers) {
     sendCommand(ZADD, argsArray);
   }
 
-  public void zadd(final byte[] key, final Map<byte[], Double> scoreMembers,
-      final ZAddParams params) {
+  public void zadd(final byte[] key, final Map<byte[], Double> scoreMembers, final ZAddParams params) {
```

Comment:
@HeartSaVioR is this format change consequence of `make format`?. If not, every time we run `make format` again it will revert this.